### PR TITLE
Change part-label to part-signifier

### PR DESCRIPTION
**Pull request by kausar007**

### CODE REVIEW

**Comment by mojavelinux** [comment]
Nice catch! Thanks!

File: docs/modules/sections/pages/part-numbers-and-labels.adoc [hunk 1]
```diff
@@ -28,15 +28,15 @@ Part I. Defensive Operations
...
+== Customize book part signifiers
 
 The label _Part_ is automatically added to the beginning of the part titles when the `partnums` attribute is set.
-You can modify the label by defining the `part-label` attribute in the header of your book.
+You can modify the label by defining the `part-signifier` attribute in the header of your book.
 
 [source]
 ----
-:part-label: Component
+:part-signifier: Component
```
Comment (mojavelinux) [inline docs/modules/sections/pages/part-numbers-and-labels.adoc:39]: Can you remove the trailing space on this line?

**Review by mojavelinux — CHANGES_REQUESTED** [review]

File: docs/modules/sections/pages/part-numbers-and-labels.adoc (hunk 1 above)
Comment (kausar007) [reply docs/modules/sections/pages/part-numbers-and-labels.adoc]: Done. Thanks

**Review by mojavelinux — APPROVED** [review]

**Comment by mojavelinux** [comment]
Thanks!

**Pull request merged**